### ISSUE TITLE
ramp_for_gutenberg_load_gutenberg only respects the most recent $criteria

### ISSUE THREAD

**Issue by justnorris** [Automattic/gutenberg-ramp#33]
Only the most last `ramp_for_gutenberg_load_gutenberg` function call will be effective.

For example: This will only enable ramp for Post ID 157

```
ramp_for_gutenberg_load_gutenberg( [ 'post_types' => [ 'post', 'page' ] ] );
// some
// code
// in
// between
ramp_for_gutenberg_load_gutenberg( [ 'post_ids' => [ 157 ] ] );
```

**Comment by brettshumaker** [comment]
I agree that this _could_ lead to a confusing situation for a user. What about adding a dismissible admin notice if we detect (via `$RFG->active`) that the function has already been called once?

I think a message somewhere would be a big help to someone trying to figure out whey _their_ function call isn’t working as expected. We could add a filter to allow developers to turn off the notice in case someone comes up with a scenario (which I'm failing to think of) where they _need_ to call the function twice.

Happy to make a PR if this approach sounds reasonable. 🙂

**Comment by justnorris** [comment]
I think that an admin message would add a bit of unnecessary overhead by making Ramp tell the user that there might be a problem instead of just fixing the problem. All this issue needs to be fixed are a couple array merges.

Maybe your time is better spent making a PR that fixes the problem instead 😉 ?

**Comment by brettshumaker** [comment]
Makes sense.

Challenge accepted. 🙂

**Comment by brettshumaker** [comment]
I've got a PR (#45) to handle this - it will merge the values of `post_ids` and `post_types` but what should we do with `load`? Currently, how my code works will just take the last value passed through `load` and use that. But using `load` will override anything set in `post_ids` and `post_types`.

That means that someone can hook into `admin_init` at priority `4` and block/load Gutenberg for everything no matter what has been set before. I don't see this being a _huge_ problem right now, but if RFG were to make it into core, it might make sense to think about deprecating `load`.

What are your thoughts on this @justnorris @mattoperry ?

**Comment by justnorris** [comment]
Fixed in #74
